Minor changelog copyediting (#49453)
CleverRaven/Cataclysm-DDA: data/changelog.txt

Allow vehicles with autopilot to follow you.
Monster factions can unconditionally hate other factions.
Allows mutations to cast spells via enchantments.
Add mining multi-activity for NPCs and players.
Basecamp job rework - wandering/sitting/job priorities.
Auto eat and drink from auto-zones during long activities or while waiting.

> Add flyable helicopters.
> Allow towing vehicles.
> Allow specfying height and age in character creation.

Reviewer: ```suggestion
Adds flyable helicopters.
Allows towing vehicles.
Allows specifying height and age in character creation.
```

Enables nested containers.
Adds support for conducts (self-imposed play constraints tracked by the game).
Custom achievement/conduct requirement descriptions.
Enables mouse input for inventory.
Worn gloves count as unarmed weapons when no weapon is held.
Any corpse can rise again as a zombie.
Added to autodocs the ability to stop bleeding, disinfect wounds, and inject antibiotics in case of infected wounds or tetanus.
Allow randomizing scenario and profession in new character menu.

> Add actions to workout and do exercise.

Reviewer: ```suggestion
Adds actions to workout and do exercises.
```

Allows nanofabricator templates to contain recipes from any item group.
Adds different bleeding types for different species.
Adds bleeding butchery action.

> Add autodoc installation programs that almost guarantee flawless installation of supported bionics.

Reviewer: ```suggestion
Adds autodoc installation programs that almost guarantee flawless installation of supported bionics.
```

Adds keycode support to uilist.
Allows attempting to disable robots by 'e'xamining them.
Warns if a friendly creature or non-hostile NPC enters the line of fire.

> Added a satiety column to the eat menu, showing how many calories the food will have for how filling it is.

Reviewer: ```suggestion
Adds a satiety column to the eat menu, showing how many calories the food will have for how filling it is.
```

Shows crafting failure rates due to missing proficiencies.
Displays contained items like 'water (plastic bottle)' instead of as 'plastic bottle of water'.
Keeps track of preferred aiming mode.

> Add Chibi Ultica tileset, a mix of MSX character and monster sprites with Ultica terrain, vehicles, and items.
> Brings back MSX+ Tileset.

Reviewer: ```suggestion
Adds the Chibi Ultica tileset, a mix of MSX character and monster sprites with Ultica terrain, vehicles, and items.
Brings back the MSX+ Tileset.
```

Shows weariness and activity levels in sidebar panels.
Warns before crafting armor or clothing you won't be able to wear.

## Mods:
[Aftershock] Adds autodoc tiers and augmentation clinics.
[Aftershock] Megacity-oriented overmap generation adjustments
[Aftershock] Adds new Melee oriented bionics.
[Aftershock] Replaces Vanilla autodocs with Aftershock Average Autodocs.
[Aftershock] Adds an astrobiology lab.
[Aftershock] Adds a design document.
[Aftershock] Adds new laser and plasma pistols.
[Aftershock] Adds Formless Ruins - a new aboveground, low-risk dungeon type.
[Blazemod] Overhauls Blazemod and splits it into separate mods.
[Dark Skies Above] DSA is reborn as an actively-developed, total-conversion mod.
[DinoMod] Adds location variants with dinosaurs.
[DinoMod] Adds Acrocanthosaurus and overhauls dino baby code.
[DinoMod] Adds monsters 'Scutellosaurus', 'Maiasaura', and 'Coelophysis'.
[DinoMod] Adds monsters 'Pachyrhinosaurus', 'Pentaceratops', and 'Torosaurus'.
[DinoMod] Organizes dinos into three main factions with defined inter-faction behavior.
[DinoMod] Rebalances Jurassic spawns and monsters for realism and fixes predator in-fighting.

> [DinoMod] Cap monsters at 1000 kg so butchering kind of works.

Reviewer: ```suggestion
[DinoMod] Caps monsters at 1000 kg so butchering kind of works.
```

[DinoMod] Adds monsters 'Brontosaurus', 'Alamosaurus', and 'Brachiosaurus'.
[DinoMod] Converts 'dinoexhibit' to use templates, adds a roof, and adds a baby dino variant.
[DinoMod] Ankylosaurs Abound: Adds Dyoplosaurus, Nodosaurus, and Edmontonia.
[DinoMod] Adds Amargasaurus and Kosmoceratops as well as variants.
[DinoMod] Adds mod-specific mutation trees Stego and Tyrant.
[Magiclysm] Adds spell level and number of spells known achievements.
[Magiclysm] Manatouched mutation tree.
[Magiclysm] Adds polymorph spells so you can transform your foes.
[Magiclysm] Adds a spell component system and more-powerful spells to use it.
[Magiclysm] Adds a new mutation line: Black Dragon.

> [Magiclysm] Adjust fog frequency in black dragon lair plus other (minor) map adjustments.

Reviewer: ```suggestion
[Magiclysm] Adjusts fog frequency in black dragon lair plus other (minor) map adjustments.
```
